src/x509_str.c

```diff
@@ -973,9 +975,7 @@ int wolfSSL_X509_STORE_set_flags(WOLFSSL_X509_STORE* store, unsigned long flag)
     if (store == NULL)
         return WOLFSSL_FAILURE;
 
-    if ((flag & WOLFSSL_CRL_CHECKALL) || (flag & WOLFSSL_CRL_CHECK)) {
-        ret = wolfSSL_CertManagerEnableCRL(store->cm, (int)flag);
-    }
+    ret = wolfSSL_CertManagerEnableCRL(store->cm, (int)flag);
```

Comment:
Is it safe to make this change, considering the new behavior only applies when OPENSSL_COMPATIBLE_DEFAULTS is defined?  Shouldn't this also be gated behind OPENSSL_COMPATIBLE_DEFAULTS?

Reply:
The `OPENSSL_COMPATIBLE_DEFAULTS` flag is unnecessary here because `wolfSSL_CertManagerEnableCRL()` does not utilize the provided <flag> when `OPENSSL_COMPATIBLE_DEFAULTS` is not defined.

Reply:
Before this change, wolfSSL_CertManagerEnableCRL is only being called if either WOLFSSL_CRL_CHECKALL or WOLFSSL_CRL_CHECK is set.  But after this change, wolfSSL_CertManagerEnableCRL is always being called.  In the case where OPENSSL_COMPATIBLE_DEFAULTS is not defined, calling wolfSSL_CertManagerEnableCRL will enable the CRL, so this is leading to a behavior change for that case.  The function X509_STORE_set_flags is used for other X509 flags unrelated to CRL, and it is not expected to enable the CRL in this case.

Is there a reason you need to remove the flag check?  If I'm understanding right, this should work fine in both cases with this check kept.